### ISSUE TITLE
Website Patreon not functioning correctly

### ISSUE THREAD

**Issue by ShadowMan44** [Feodor2/Mypal#394]
On Patreon, I cannot send direct messages to people. I cannot seem to click on the text field to start typing.
What's the cause of this exactly? Is it possible to fix at all?

**Comment by Feodor2** [comment]
Assuming this is www.patreon.com and it requires signup, well i try signup and check.

**Comment by ShadowMan44** [comment]
Yes, you have to sign up and then you have to actually subscribe to a patreon creator in order to message them.
